fastp/0.23.2/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:bionic as app
+
+ARG FASTP_VER="0.23.2"
+
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="1"
+LABEL software="Fastp"
+LABEL software.version=$SOFTWARENAME_VER
+LABEL description="An ultra-fast all-in-one FASTQ preprocessor"
+LABEL website="https://github.com/OpenGene/fastp"
+LABEL license="https://github.com/OpenGene/fastp/blob/master/LICENSE"
+LABEL maintainer="Idowu Olawoye"
+LABEL maintainer.email="olawoyei0303@run.edu.ng"
+
+RUN apt-get update && apt-get install -y \
+ wget
+
+RUN mkdir fastp && \
+ cd fastp && \ 
+ wget http://opengene.org/fastp/fastp.0.23.1 && \
+ mv fastp.0.23.1 fastp && \
+ chmod a+x ./fastp && \
+ mkdir /data
```

Comment:
Could you replace the hard-coded version with your variable? It helps when updating the file for future releases

Something like the following:

```
RUN mkdir fastp && \
 cd fastp && \ 
 wget http://opengene.org/fastp/fastp.${FASTP_VER} && \
 mv fastp.${FASTP_VER} fastp && \
 chmod a+x ./fastp && \
 mkdir /data
 ```